api/v1beta1/verticadb_types.go

```diff
@@ -1104,10 +1104,11 @@ func MakeVDB() *VerticaDB {
 
 // MakeVDBForHTTP is a helper that constructs a VerticaDB struct with http enabled.
 // This is intended for test purposes.
-func MakeVDBForHTTP() *VerticaDB {
+func MakeVDBForHTTP(name string) *VerticaDB {
```

Comment:
Suggest using a different name:
```suggestion
func MakeVDBForHTTP(httpServerTLSSecretName string) *VerticaDB {
```